server/webserver.go

```diff
@@ -76,6 +76,18 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 	mux := http.DefaultServeMux
 	http.DefaultServeMux = http.NewServeMux()
 
+	mux.HandleFunc("/signMessage", func(w http.ResponseWriter, r *http.Request) {
+		if err := r.ParseForm(); err != nil {
+			glog.Errorf("Parse Form Error: %v", err)
```

Comment:
Should return if we hit an error here.

[mustHaveFormParams](https://github.com/livepeer/go-livepeer/blob/a9e6dd764ee179db69309618917d176b5cba26fc/server/handlers.go#L28) might be helpful here - see this [example usage](https://github.com/livepeer/go-livepeer/blob/master/server/webserver.go#L1100)).